test-case/verify-pcm-list.sh

```diff
@@ -26,16 +26,15 @@ OPT_HAS_ARG['t']=1         OPT_VAL['t']="${TPLG:-}"
 func_opt_parse_option "$@"
 tplg=${OPT_VAL['t']}
 
-tplg_path=`func_lib_get_tplg_path "$tplg"`
-[[ "$?" != "0" ]] && die "No available topology for this test case"
+tplg_path=`func_lib_get_tplg_path "$tplg"`|| die "No available topology for this test case"
```

Comment:
You can have a new line after `||` if you prefer shorter lines. No need to escape it.

Reply:
OK , I will try shellcheck for better code style :)